packages/airnode-adapter/e2e/index.ts

```diff
@@ -235,27 +217,31 @@ describe('Extraction, encoding and simple on chain decoding', () => {
 
   it('decodes string32 encoded by the adapter package', async () => {
     const { parseBytes32String, arrayify } = ethers.utils;
-    const encodedBytes = extractAndEncodeResponse(apiResponse, { _type: 'string32', _path: 'string' }).encodedValue;
+    const encodedBytes = extractAndEncode({ _type: 'string32', _path: 'string' });
 
     const fromContract = await testDecoder.decodeString32(encodedBytes);
     const decoded = parseBytes32String(arrayify(fromContract)).toString();
     expect(decoded).to.equal(apiResponse.string);
   });
 
+  it('decodes multiple params encoded by the adapter package', async () => {
+    const decoded = await testDecoder.decodeMultipleParameters(
+      extractAndEncode({ _type: 'string,uint256,address', _path: 'big.string,float,address', _times: ',10000,' })
```

Comment:
Note that I am not using `null` to specify that a value should not be multiplied, but rather empty string.

This is because `_path` is optional as well - and that returns the whole API response. Also, `null` in path would be ambiguous (do you want key or the whole response?). Empty string solves this.

Reply:
Does this also apply for `_path`, i.e. is `big.string,` valid as a `_path` value?

Reply:
Yes.